src/j1client.d.ts

```diff
@@ -30,7 +30,7 @@ declare module "@jupiterone/jupiterone-client-nodejs" {
   export default class JupiterOneClient {
     constructor(options: JupiterOneClientOptions);
 
-    init(): Promise<JupiterOneClient>;
+    init(httpLink?: HttpLink): Promise<JupiterOneClient>;
```

Comment:
I think it's worthwhile to consider another approach to making the client class testable, instead of changing the public interface and exposing the implementation details (the `HttpLink`). Polly.js is pretty great for controlling the HTTP subsystem that I suppose Apollo must be using. This way, you could test that the client does what you expect without knowing how it makes HTTP requests.